docs/guide/defining_an_energy_network.rst

```diff
@@ -304,33 +304,26 @@ production and/or space heat production. Links allow this sharing to be possible
 defined already for buses excel sheet. ``invest_base`` and ``invest_cap`` parameters (defined in the transformers sheet
 section) are only relevant for space heating links in the present stage of development.
 
-.. image:: ./resources/input_excel_links.PNG
```

Comment:
could you delete the old image input_excel_links.PNG file, if not done already?

Reply:
done